package.json

```diff
@@ -44,7 +44,7 @@
     "start": "yarn storybook:start",
     "storybook:start": "start-storybook -p 9002",
     "docs:publish": "node scripts/publish-docs.js",
-    "i18n:build": "mc-scripts extract-intl --build-translations --output-path=$(pwd)/i18n 'src/components/**/!(*.spec).js'",
```

Comment:
I didn't see the point of keeping "build-translations", since we aren't using tx here, so we always need to build translations. If we start using TX, we can add it back.